src/youtube-player/README.md

```diff
@@ -20,36 +20,34 @@ If your video is found at https://www.youtube.com/watch?v=PRQCAL_RMVo, then your
 
 ```typescript
 // example-module.ts
-import {NgModule, Component, OnInit} from '@angular/core';
-import {YouTubePlayerModule} from '@angular/youtube-player';
+import {NgModule, Component} from '@angular/core';
+import {
+  YouTubePlayerModule,
+  YOUTUBE_PLAYER_CONFIG,
+  YouTubePlayerConfig,
+} from '@angular/youtube-player';
 
 @NgModule({
   imports: [YouTubePlayerModule],
+  // Optionally tells the `youtube-player` component to automatically load
+  // the YouTube iframe API. Omit this if you plan to load the API yourself.
+  providers: [{
```

Comment:
@jelbourn I would appreciate your feedback on a couple of things that I was considering when implementing it:

1. How the config should be provided: through an injection token like here or through something like `YouTubePlayerModule.withConfig({ loadApi: true })`. I went with the former, because it's a bit clearer what is going on, but I can see an argument for the latter, because it is a bit cleaner and I expect that the most common use case would be to let the component load the API automatically. I wouldn't mind changing it to use the second approach.
2. Whether `loadApi` should be turned on by default. I decided against it, because it does add an external script to people's apps and I figured that's something we'd want them to opt into, but it does make it slightly harder to consume.